src/service/email.service.js

```diff
@@ -0,0 +1,60 @@
+import 'dotenv/config';
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: false, // true for port 465, false for other ports
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+export function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+}
+
+function sendActivation(email, token) {
+  const href = `${process.env.CLIENT_HOST}/activate/${email}/${token}`;
```

Comment:
Ensure that the activation link is properly encoded to prevent issues with special characters in the email or token.